pkg/apis/serving/v1beta1/configmap_test.go

```diff
@@ -87,10 +87,29 @@ func TestNewIngressConfig(t *testing.T) {
 	g := gomega.NewGomegaWithT(t)
 	clientset := fakeclientset.NewSimpleClientset(&v1.ConfigMap{
 		ObjectMeta: metav1.ObjectMeta{Name: constants.InferenceServiceConfigMapName, Namespace: constants.KServeNamespace},
+		Data: map[string]string{
+			IngressConfigKeyName: `{
+				"ingressGateway": "knative-serving/knative-ingress-gateway",
+				"ingressService": "test-destination",
+				"localGateway": "knative-serving/knative-local-gateway",
+				"localGatewayService": "knative-local-gateway.istio-system.svc.cluster.local",
+				"urlScheme": "https",
+				"ingressDomain": "example.com",
+				"additionalIngressDomains": ["other-example.com", "other-example-second.com"]
+			}`,
+		},
 	})
 	ingressCfg, err := NewIngressConfig(clientset)
 	g.Expect(err).Should(gomega.BeNil())
 	g.Expect(ingressCfg).ShouldNot(gomega.BeNil())
+
+	g.Expect(ingressCfg.IngressGateway).To(gomega.Equal("knative-serving/knative-ingress-gateway"))
```

Comment:
some of these values are hard coded multiple times like "knative-serving/knative-ingress-gateway", is there a constant you can refer to or do you think it would be helpful to make them?

Reply:
I will make it neater.